### PR TITLE
KFLUXINFRA-1163: Updating Host Config for MPC

### PR DESCRIPTION
**Pull request by manish-jangra**
With Addition of kflux-prd-rh02 in the list, this pull request will retructure the production overlay by further creating base, stone-prd-rh01 and kflux-prd-rh02 overlays.

Earlier Pull that was closed due to merge conflicts -
- https://github.com/redhat-appstudio/infra-deployments/pull/5097

### CODE REVIEW

**Comment by bamachrn** [comment]
Hi @manish-jangra we are having a lot of issues with the IBM archs recently. lets add the AWS infra for now, then we will add IBM archs once we have them running and tested with prod-rh01

**Comment by manish-jangra** [comment]
do-not-merge/work-in-progress

**Comment by manish-jangra** [comment]
> Hi @manish-jangra we are having a lot of issues with the IBM archs recently. lets add the AWS infra for now, then we will add IBM archs once we have them running and tested with prod-rh01

I'll remove the IBM Configuration from host-config

File: components/multi-platform-controller/production/stone-prd-rh01/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (mshaposhnik) [inline components/multi-platform-controller/production/stone-prd-rh01/kustomization.yaml:1]: just for my information - is there any particular reason some servers starts from `stone-*-*` and some are `kflux-*-*` or it is just historical heritage?

**Pull request merged**